channels/ru.m3u

```diff
@@ -361,9 +351,9 @@ http://188.40.68.167/russia/komediya/playlist.m3u8
 http://cdn-01.bonus-tv.ru:80/konnyimir/playlist.m3u8
 #EXTINF:-1 tvg-id="Krasnayaliniya.ru" tvg-name="Красная линия" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="General",Красная линия (480p)
 https://kprf-htlive.cdn.ngenix.net/live/stream_high/playlist.m3u8
-#EXTINF:-1 tvg-id="KrasnogorskTV.ru" tvg-name="Красногорск ТВ" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",Красногорск ТВ (480p)
+#EXTINF:-1 tvg-id="KrasnogorskTV.ru" tvg-name="Красногорск ТВ" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",Красногорск ТВ (480p) [Not 24/7]
 http://live-krtv.cdnvideo.ru/krtv/krtv.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="KrikTV.ru" tvg-name="Крик-ТВ" tvg-country="RU" tvg-language="Russian" tvg-logo="http://ricekb.ru/img/logo.png" group-title="",Крик-ТВ (720p)
+#EXTINF:-1 tvg-id="KrikTV.ru" tvg-name="Крик-ТВ" tvg-country="RU" tvg-language="Russian" tvg-logo="http://ricekb.ru/img/logo.png" group-title="",Крик-ТВ (720p) [Offline]
 https://hls-rictv.cdnvideo.ru/rictv/rictv.sdp/playlist.m3u8
```

Comment:
https://strm.yandex.ru/kal/krik_tv/krik_tv0.m3u8

Reply:
That link works.

Reply:
Ah yeah, you got it before I did. Gonna remove my comment.

Reply:
Still doesn't work for me:
![image](https://user-images.githubusercontent.com/7253922/128776027-8e8cc578-2be1-45d1-a7f1-5f7a601e241c.png)

Reply:
I meant the link Dum4G posted. It works, the other doesn't.

Reply:
Oh I see. I've already added a new link.